README.md

```diff
@@ -244,7 +244,7 @@ tasks.koverCollectReports {
 
 ```groovy
 tasks.koverCollectReports {
-    into "$buildDir/my-reports-dir"
+  outputDir.set(layout.buildDirectory.dir("my-reports-dir") )
```

Comment:
What's wrong with `$buildDir/my-reports-dir`?

Reply:
`outputDir` has type `DirectoryProperty` not `String`